docs/design-ux/file-types.md

```diff
@@ -0,0 +1,54 @@
+# First party support
+
+The following files will receive first party support on all platforms:
+
++ Plain Text
++ Drawings
++ Todo lists
+
+These are the set of file types (for now) that we will author UI elements for.
+
+# Second party support
+
+For arbitrary file types we have a few options.
+
+## [FUSE](https://en.wikipedia.org/wiki/Filesystem_in_Userspace)
```

Comment:
FUSE on linux may be a non-issue because it comes pre-installed / compiled into the compiler.